src/mui/field/ImageField.js

```diff
@@ -11,6 +11,10 @@ const styles = {
     },
 };
 
+const clear = {
```

Comment:
You're right. In last version, we introduced an `ImageInputPreview` component. The `float: left;` should be here instead. This way, it wouldn't cause trouble on `<ImageField />` used in datagrids for instance.